src/MotionControl.c

```diff
@@ -1436,6 +1430,34 @@ BOOL Ros_MotionControl_StartMotionMode(MOTION_MODE mode)
         }
     }
 
+    // Check if in continous cycle mode
+    if (!Ros_Controller_IsContinuousCycle())
+    {
+        // set the cycle mode to auto if not currently
+        MP_CYCLE_SEND_DATA sCycleData;
+        bzero(&sCycleData, sizeof(sCycleData));
+        bzero(&rData, sizeof(rData));
+        sCycleData.sCycle = 3;  // Auto
```

Comment:
M+ does not define any named constant(s) for this?

Reply:
It does not